sormas-api/src/main/java/de/symeda/sormas/api/utils/CSVUtils.java

```diff
@@ -44,7 +46,67 @@ public static CSVReader createCSVReader(Reader reader, char separator, LineValid
 	}
 
 	public static CSVWriter createCSVWriter(Writer writer, char separator) {
-		return new CSVWriter(writer, separator, CSVWriter.DEFAULT_QUOTE_CHARACTER, CSVWriter.DEFAULT_ESCAPE_CHARACTER, CSVWriter.DEFAULT_LINE_END);
+		return new SafeCSVWriter(writer, separator, CSVWriter.DEFAULT_QUOTE_CHARACTER, CSVWriter.DEFAULT_ESCAPE_CHARACTER, CSVWriter.DEFAULT_LINE_END);
 	}
 
+	/**
+	 * Extension of the {@link CSVWriter} which sanitizes each element of the CSV to prevent CSV Injection
+	 * @see  <a href="https://owasp.org/www-community/attacks/CSV_Injection">CSV Injection</a>
+	 */
+	public static class SafeCSVWriter extends CSVWriter {
+
+		private static final String FORMULA_PREFIX = "'";
+
+		private final Pattern formulaPattern = Pattern.compile("(?s)^[+=@-].*", Pattern.MULTILINE);
+
+		public SafeCSVWriter(Writer writer, char separator, char quotechar, char escapechar, String lineEnd) {
+			super(writer, separator, quotechar, escapechar, lineEnd);
+		}
+
+		@Override
+		public void writeNext(String[] nextLine, boolean applyQuotesToAll, Appendable appendable) throws IOException {
```

Comment:
@vidi42 This code has been taken and extended from CSVWriter, right? From a performance standpoint that's a good choice, but I'd suggest to clearly mark that this is copied and which part is custom, so we can later update this to any changes made in CSVWriter. 
I'd also keep the separate appendQuoteCharacterIfNeeded method.